catalyst_voices/lib/pages/account/delete_keychain_dialog.dart

```diff
@@ -0,0 +1,156 @@
+                  Text(
+                    'Delete Keychain?',
+                    style: Theme.of(context).textTheme.titleLarge,
+                  ),
+                  const SizedBox(height: 48),
+                  Text(
+                    '''
+Are you sure you wants to delete your  Catalyst Keychain from this device?''',
+                    style: Theme.of(context).textTheme.titleMedium,
+                    textAlign: TextAlign.center,
+                  ),
+                  const SizedBox(height: 32),
+                  Wrap(
+                    children: [
+                      VoicesAssets.icons.exclamation.buildIcon(
+                        color: Theme.of(context).colors.iconsError,
+                      ),
+                      Padding(
+                        padding: const EdgeInsets.only(top: 2, left: 8),
+                        child: Text(
+                          '''
+Make sure you have a working Catalyst 12-word seedphrase!''',
+                          style: Theme.of(context).textTheme.titleSmall,
+                        ),
+                      ),
+                    ],
+                  ),
+                  const SizedBox(height: 8),
+                  Text(
+                    '''
+Your Catalyst account will be removed, this action cannot be undone!''',
+                    style: Theme.of(context).textTheme.bodyMedium,
+                    textAlign: TextAlign.center,
+                  ),
+                  const SizedBox(height: 48),
+                  Text(
+                    'To avoid mistakes, please type ‘Remove Keychain’ below.',
+                    style: Theme.of(context).textTheme.titleSmall,
+                  ),
+                  const SizedBox(height: 24),
+                  Wrap(
+                    direction: Axis.vertical,
+                    children: [
+                      Text(
+                        'Confirm removal',
+                        style: Theme.of(context).textTheme.titleSmall,
+                      ),
+                      const SizedBox(height: 2),
+                      SizedBox(
+                        width: 300,
+                        child: VoicesTextField(
+                          controller: _textEditingController,
+                          decoration: VoicesTextFieldDecoration(
+                            errorText: _errorText,
+                            filled: true,
+                            fillColor: Theme.of(context)
+                                .colors
+                                .elevationsOnSurfaceNeutralLv1White,
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                  const SizedBox(height: 24),
+                  Wrap(
+                    children: [
+                      VoicesFilledButton(
+                        backgroundColor: Theme.of(context).colors.iconsError,
+                        onTap: () async {
```

Comment:
extract this onTap function into private one in **_DeleteKeychainDialogState** 

smth like this
```dart
VoicesFilledButton(
onTap: _verygoodname
);

Future<void> _verygoodname(){
                          if (_textEditingController.text ==
                              'Remove Keychain') {
                            // TODO(Jakub): remove keychain
                            Navigator.of(context).pop();
                            await VoicesDialog.show<void>(
                              context: context,
                              builder: (context) {
                                return const KeychainDeletedDialog();
                              },
                            );
                          } else {
                            setState(() {
                              _errorText = 'Text incorrect';
                            });
                          }
}
```